containers/domains/DomainName.js

```diff
@@ -14,7 +14,12 @@ const DomainName = ({ domain }) => {
 
     return (
         <>
```

Comment:
Since you are wrapping the variable with a `<span>` you should be able to remove the `React.Fragment`: `<>`.

Reply:
Good catch, fixed via https://github.com/ProtonMail/react-components/pull/522/commits/7aa62f86ef3f3d3acec9c86999ece30aeb7c7cde